pkg/config/sync.go

```diff
@@ -287,6 +287,9 @@ func mergeConfig(from, to *controller.OperatorConfiguration) {
 				to.Workspace.ServiceAccount.DisableCreation = pointer.BoolPtr(*from.Workspace.ServiceAccount.DisableCreation)
 			}
 		}
+		if from.Workspace.ServiceAccountTokens != nil {
+			to.Workspace.ServiceAccountTokens = from.Workspace.ServiceAccountTokens
+		}
```

Comment:
Do we want to support a more complex merge here, or is overwriting sufficient? I.e. do we want to support the case where the default DWOC specifies one token and the override specifies second one?

Not supporting is probably sufficient for now.

Reply:
I also think not supporting a complex merge is fine for now. 

@l0rd do you happen to have any input on this?